IDE/STM32Cube/default_conf.ftl

```diff
@@ -275,6 +275,13 @@ extern ${variable.value} ${variable.name};
     #define NO_SESSION_CACHE
 #endif
 
+/* Post Quantum */
+#if defined(WOLF_CONF_PQC) && WOLF_CONF_PQC == 1
+    #define HAVE_PQC
+    /* #define HAVE_LIBOQS */
```

Comment:
This template will be pulled into the next STM32 wolfSSL cube pack. Please add some comments here about where to find these.

Reply:
> Please add some comments here about where to find these.
When you say "these", what you mean?

Reply:
oh..you mean pqm4. ok.